src/components/Videos.vue

```diff
@@ -123,6 +125,23 @@ export default {
 			// reset and show poster after play
 			this.$el.autoplay = false
 			this.$el.load()
+		},
+
+		saveVolume() {
+			localStorage.videoVolume = this.$el.volume
+		},
+
+		setVolume() {
+			if (localStorage.videoVolume) {
+				this.$el.volume = localStorage.videoVolume
+			} else {
+				this.$el.volume = 0.5
```

Comment:
So we set the default at .5?

Reply:
Yes. I think 50% may be more appropriate than the default 100%. It's probably subjected to personal opinion though.

Reply:
If the defaukt is 100%, I think we should keep it like that. That's what people expect (youtube, facebook, imgur, reddit... all have the volume at 100% by default :) )

Reply:
You're right. I have removed that part in the new commit.